tests/kernel/boot_ltp.pm

```diff
@@ -109,7 +109,8 @@ EOF
 
         # emulate $LTPROOT/testscripts/network.sh
         assert_script_run('curl ' . data_url("ltp/net.sh") . ' -o net.sh', 60);
-        assert_script_run('sh ./net.sh');
+        assert_script_run('chmod 755 net.sh');
+        assert_script_run('. ./net.sh');
```

Comment:
not something really critical but since you touching this anyway - don't you think that script which meant to emulate network.sh  should have name network.sh ? Or if you afraid some kind of confusion "emulate_network.sh" ?